src/app/utils/codemirror/eslint-lint.js

```diff
@@ -336,7 +337,11 @@ function getSeverity(error) {
 
 function eslintValidate(text) {
   if (!linter) return [];
-  return linter.verify(text, defaultConfig);
+  const saveRequire = window.require;
+  window.require = eslintRequire;
+  const result = linter.verify(text, defaultConfig);
+  window.require = saveRequire;
+  return result;
```

Comment:
When switching from Monaco to CodeEditor, `window.require` refers to Monaco's loader, clashing with `linter.verify()` which calls `require()` for some reason. So we save eslint's require (see below) and use it every time we call `linter.verify()`.